scripts/docker-smoke.sh

```
@@ -0,0 +1,92 @@
+"nuxt-self-bun|--frontend nuxt --backend self --runtime none --database postgres --orm drizzle --db-setup docker --web-deploy docker --server-deploy none --api orpc --auth better-auth --payments none --addons none --examples none --package-manager bun|3001|0|0"
+)
+
+i=0
+for entry in "${TESTS[@]}"; do
+  i=$((i+1))
+  IFS='|' read -r name flags webint has_server spa <<< "$entry"
+  WEB_PORT=$((4100+i)); SRV_PORT=$((4200+i)); DB_PORT=$((4300+i))
+  dir="$SCRATCH/$name"
+  echo "=== [$i/${#TESTS[@]}] $name ===" | tee -a "$RESULTS"
+  rm -rf "$dir"
+  cd "$SCRATCH"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)